### PR TITLE
Embed videos directly into Readme

### PR DESCRIPTION
**Pull request by pushkaraj2007**
## Changes I made:
- Moved demo videos section
- Embedded videos directly into Readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,18 @@
...
+
+Check out the first demo video of Kaguya: 
+
+https://github.com/pushkaraj2007/kaguya/assets/107422421/3000d44a-9c89-4c0f-99bf-8df131aa71ee
+
+Check out the second demo video of Kaguya:
+
+https://github.com/pushkaraj2007/kaguya/assets/107422421/6b180222-6ac5-4d42-9b40-7d48fd37f961
+
+You can check out a second demo of Kaguya in action on Twitter: [Second Demo Video](https://twitter.com/ykdojo/status/1670848611532562433)
```
Comment (ykdojo) [inline README.md:15]: Looks like this is repetitive

File: README.md [hunk 2]
```diff
@@ -2,6 +2,18 @@
...
 
+## More About Kaguya
+
+Check out the first demo video of Kaguya: 
+
+https://github.com/pushkaraj2007/kaguya/assets/107422421/3000d44a-9c89-4c0f-99bf-8df131aa71ee
+
+Check out the second demo video of Kaguya:
+
+https://github.com/pushkaraj2007/kaguya/assets/107422421/6b180222-6ac5-4d42-9b40-7d48fd37f961
```
Comment (ykdojo) [inline README.md:13]: can we actually swap the order and show the second video first? I think it'll be more impactful that way.

also can we reduce the repetitiveness in the wording of these two sentences?

**Comment by ykdojo** [comment]
Thank you for this! I added a few comments

**Review by ykdojo — CHANGES_REQUESTED** [review]

**Comment by pushkaraj2007** [comment]
Hey @ykdojo 

I have Swapped the demo videos and also removed the extra line.

I'm not sure about the repetitiveness of the words you are talking about. Can you explain it a bit more? 

Should we write it as:

Here is the first demo video of Kaguya:
and
Here is the second demo video of Kaguya:

**Comment by ykdojo** [comment]
It's not that big of a deal, but it's just that those two sentences have some repetitiveness, namely: "Here's the 1st/2nd demo video of Kaguya:"

**Comment by pushkaraj2007** [comment]
@ykdojo 

What if we only keep "Here are the demo videos of Kaguya:"

And show both of the videos?

**Comment by ykdojo** [comment]
That sounds good! Thank you.

**Comment by pushkaraj2007** [comment]
Let me do it, real quick!

**Comment by pushkaraj2007** [comment]
Let me do it, real quick!

**Comment by pushkaraj2007** [comment]
I have made the changes. Can you please check it now?

**Comment by ykdojo** [comment]
Looks good! Thank you

**Pull request merged**